pandas/tests/indexes/datetimes/test_timezones.py

```diff
@@ -327,6 +327,16 @@ def test_tz_convert_unsorted(self, tzstr):
     # -------------------------------------------------------------
     # DatetimeIndex.tz_localize
 
+    def test_tz_localize_utc_copies(self):
+        times = ["2015-03-08 01:00", "2015-03-08 02:00", "2015-03-08 03:00"]
+        index = DatetimeIndex(times)
+
+        res = index.tz_localize("UTC")
```

Comment:
Probably good to test with `utc_fixture`.

Also based on the code code `None` also returns a copy now too (and should be tested)?

Reply:
sure.  will use the fixture in the index tests and do the None just once in the tslibs tests